src/components/editors/ExportEditor.js

```diff
@@ -31,9 +31,8 @@ const ExportEditor = () => {
 
   const resetEditor = () => {
     window.localStorage.clear()
-    loadStorage()
-    loadCss()
     setSettings( false )
+    window.location.reload()
```

Comment:
I'm still a little iffy about this, but it's acceptable I suppose.

Reply:
Can try using this https://www.npmjs.com/package/use-force-update